TypeScript.YAML-tmLanguage

```diff
@@ -241,6 +241,8 @@ repository:
     end: (?=$|[,);\}\]]|//)|(?==[^>])|(?<=[\}>\]\)]|[a-zA-Z_$])\s*(?=\{)
     patterns:
     - include: '#type'
+    - include: '#string'
```

Comment:
We're also thinking about having string literal types eventually too, so it will also make actual sense in the future. That also means that the following is going to be a thing as well:

```TypeScript
type State = "Ready" | "Running" | "Recalibrating" | "Reticulating Splines" | "Returning";
```